content/pages/constitution.md

```diff
@@ -0,0 +1,313 @@
+computer science community at large.
+
+## Representation ##
+
+The Women in Computer Science Undergraduate Subcommittee aims to
+represent and support undergraduate cis and trans women, and non-binary
+individuals in computer science. Additionally, we seek to include and
+support members of all underrepresented groups in the fields of
+technology and computing, with the most varied and diverse backgrounds
+possible.
+
+## Values ##
+
+1.  Inclusiveness: The undergraduate commitee is committed to creating
+    and inclusive environment in which women of all backgrounds will
+    feel welcome.
+
+2.  Humility: In all it's activities, the committee will acknowledge and
+    respect the perspectives of those with experiences different to
+    ours, and will strive to understand and incorporate these
+    perspectives.
+
+3.  Respoect: All members of the community will be respected, despite
+    difference in opinions and values.
+
+4.  Transparency: The actions of the committee will be visible to
+    promote accountability, and all decision-making will be open and
+    well-documented.
+
+5.  Honesty: The committee will create an environment to promote honest
+    disclosure. All members should feel free speaking about any opinions
+    and feelings.
+
+6.  Technical Excellence: The committee acknowledges that it's
+    fundamental goal is to support diversity in technical work, and
+    members will strive to achieve excellence in this work themselves.
+
+# Committee Composition #
+
+1.  Only undergraduate cis- and trans-women and non-binary individuals
+    enrolled in computer science or a related program may serve on the
+    committee. The committtee will be comprised of the following roles:
+
+    -   Chair,
+
+    -   Secretary,
+
+    -   Finance Director,
+
+    -   Office Manager,
+
+    -   Systems Administrator,
+
+    -   Big CSters Director,
+
+    -   Director of Publicity,
+
+    -   Up to three additional positions as necessary, with titles
+        chosen by the committee.
+
+2.  The duties of the Chair are:
+
+    -   To chair all meetings;
+
+    -   To publicly represent the committee as the direct link to
+        external groups;
+
+    -   To represent the undergraduate committee at meetings of the WiCS
+        Committee;
+
+    -   To represent the undergraduate committee at meetings of any
+        other external organization; to which the commiittee is
+        accountable, when appropriate, such as the Mathematics Society;
+
+    -   To secure funding for the committee's endeavours in
+        collaboration with the Finance Director.
+
+3.  The duties of the Secretary are:
+
+    -   To take minutes at all meetings;
+
+    -   To call meetings;
+
+    -   To represent the undergraduate committee at meetings of the WiCS
+        Committee;
+
+    -   To build consensus among members in the decision-making process;
+
+    -   To ensure all members are comleting assigned resposibilities;
```

Comment:
"ensure" is a bit strong. Maybe "To track the completion of all duties assigned to members of the committee"?